### PR TITLE
w-json 1.1.0

### PR DESCRIPTION
**Pull request by IonicaBizau**
- Add a new line character at the end if `options.new_line` is true.

:dizzy:

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
:punch:

File: package.json [hunk 1]
```diff
@@ -24,4 +24,4 @@
     "url": "https://github.com/IonicaBizau/node-w-json/issues"
   },
   "homepage": "https://github.com/IonicaBizau/node-w-json"
-}
+}
```
Comment (PeterDaveHello) [inline package.json:27]: No end of line now?

Comment (IonicaBizau) [inline package.json:27]: That's written by [`blah`](https://github.com/IonicaBizau/node-blah). And `blah` uses `w-json`. You got it. :stuck_out_tongue_winking_eye: 

[Here](https://github.com/IonicaBizau/node-blah/blob/master/lib/index.js#L438) is should be:

``` js
WriteJson(self.paths.pack, self.pack, true, next);
```

Not a big thing, anyway. :smile:

**Pull request merged**